common/src/main/java/com/datastax/oss/simulacron/common/stubbing/PeerMetadataHandler.java

```diff
@@ -60,7 +60,9 @@
 
   static final UUID schemaVersion = java.util.UUID.randomUUID();
 
-  private static final String queryClusterName = "select cluster_name from system.local";
+  private static final Pattern queryClusterName =
+      Pattern.compile(
+          "select cluster_name from system.local( where key='local')?", Pattern.CASE_INSENSITIVE);
```

Comment:
nit: `select cluster_name from system.local( where key='local')?` => `SELECT cluster_name FROM system.local( WHERE key='local')?`

Just going for consistency with other CQL patterns (even though it looks like this one slipped through before).